packages/caravan-health/src/feescore.test.ts

```diff
@@ -0,0 +1,97 @@
+import { feesScore, feesToAmountRatio, relativeFeesScore } from "./feescore";
+import { BlockchainClient } from "@caravan/clients";
+import { Transaction } from "./types";
+import { AddressUtxos } from "./types";
```

Comment:
```suggestion
import { Transaction, AddressUtxos } from "./types";
```

Prettier should catch these for you if you have setup in your IDE.